app/assets/javascripts/admin/addon/components/admin-watched-word.js

```diff
@@ -6,7 +6,7 @@ import { action } from "@ember/object";
 import I18n from "I18n";
 
 export default Component.extend({
-  classNames: ["watched-word"],
```

Comment:
@davidtaylorhq is there going to be a problem in any kind, eg. lookup or listener, by moving this from the component to the template?

Reply:
I'm not 100% sure what you're asking here. Are you able to clarify?